packages/webpack/preset.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const { join } = require('path');
+
+module.exports = {
+  browsers: ['defaults'],
+  node: 'current',
+  locations: [],
+  basePath: '',
+  assetPath: '<basePath>',
+  buildDir: '<distDir>',
+  serverDir: join('<rootDir>', 'node_modules', '.cache', 'hops-webpack'),
```

Comment:
btw: do we break team's deployments by changing that pathname?

Reply:
I would guess we don't break anything here.
1. its a cache
2. after a build this new folder will be used (so for a new deployment the new folder would be used)
3. No one should externally rely that their artifacts are stored in a node_modules/.cache/... folder

Reply:
Yeah, confused that with the assets, needed in the browser. But those end up in a `./dist`-folder, so everything should be fine.